cinema/serializers.py

```diff
@@ -22,3 +21,60 @@
         instance.save()
 
         return instance
+
+
+class ActorSerializer(serializers.Serializer):
```

Comment:
Consider using `serializers.ModelSerializer` for `ActorSerializer` to automatically handle fields and methods based on the `Actor` model.